### PR TITLE
31 propose a new navigation

### PR DESCRIPTION
**Pull request by adlbrtbsr**
Nav menu is now shown on every page, also made some cosmetic changes

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -34,35 +36,41 @@ const App = () => {
...
-                  <MoviePage />
-                </RequireAuth>
-              }
-            />
-          </Routes>
+          <div className={"logoBar"}>
+            <img className={"logo"} src={logo} alt={"logo"} />
+            <MovieMenu />
+          </div>
+          <div className={"pageContent"}>
```
Comment (LissaGreense) [inline frontend/src/App.tsx:43]: I like this idea to put div with "pageContent" here very much! It removes the redundant code. Unfortunately right now we have duplicated dives in following pages:

- Movies
- Login
- Register

Please clean div with "pageContent" and "logoBar" from those pages and check if everything is still alright :D

File: frontend/src/components/movieMenu.tsx [hunk 2]
```diff
@@ -32,7 +33,7 @@ export default function MovieMenu() {
   }, []);
 
   const itemRenderer = (item: MenuItem) => (
-    <div className="p-menuitem-content">
+    <div className="p-menuitem-content menuItemLogoBar">
```
Comment (LissaGreense) [inline frontend/src/components/movieMenu.tsx:36]: Remove class "menuItemLogoBar", it's nor resolvable

File: frontend/src/pages/loginPage.css [hunk 3]
```diff
@@ -1,6 +1,10 @@
 .loginContainer {
   width: 30%;
   height: 35%;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```
Comment (LissaGreense) [inline frontend/src/pages/loginPage.css:7]: absoluteCenter ;)

File: frontend/src/pages/accountPage.css [hunk 4]
```diff
@@ -5,8 +5,12 @@
...
 
 .accountContainer {
-  width: 50%;
-  height: auto;
+  width: 60%;
+  height: fit-content;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```
Comment (LissaGreense) [inline frontend/src/pages/accountPage.css:13]: absolute center :)

File: frontend/src/components/movieMenu.tsx (hunk 2 above)
Comment (adlbrtbsr) [reply frontend/src/components/movieMenu.tsx]: removed

File: frontend/src/pages/calendarPage.css [hunk 5]
```diff
@@ -1,4 +1,8 @@
 .pageContentCalendar {
-  display: flex;
-  margin-top: 12%;
+  height: 50%;
+  width: 60%;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```
[the remark this answers is not in the document]
Comment (adlbrtbsr) [reply frontend/src/pages/calendarPage.css]: moved

File: frontend/src/pages/loginPage.css (hunk 3 above)
Comment (adlbrtbsr) [reply frontend/src/pages/loginPage.css]: moved

**Review by LissaGreense — APPROVED** [review]
Good job! :D

**Pull request merged**